pkg/envoy/rds/response.go

```diff
@@ -18,13 +18,6 @@ import (
 
 // NewResponse creates a new Route Discovery Response.
 func NewResponse(cataloger catalog.MeshCataloger, proxy *envoy.Proxy, discoveryReq *xds_discovery.DiscoveryRequest, cfg configurator.Configurator, _ *certificate.Manager, proxyRegistry *registry.ProxyRegistry) ([]types.Resource, error) {
-	proxyIdentity, err := envoy.GetServiceIdentityFromProxyCertificate(proxy.GetCertificateCommonName())
-	if err != nil {
-		log.Error().Err(err).Str(errcode.Kind, errcode.GetErrCodeWithMetric(errcode.ErrGettingServiceIdentity)).
```

Comment:
Looks like `errcode.ErrGettingServiceIdentity` is no longer used at all.

Reply:
done!